source/CardGames.py

```diff
@@ -156,27 +179,3 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
-
-def printHands(players: dict):
-    # Show each player's hand
-    for player in players.values():
-        print(f'{player.name}:')
-        player.showHand(True)
-        print()
```

Comment:
Its kind of weird just using print() you could instead use n\ in the print statement earlier to accomplish the same thing.

Reply:
Ok. I will modify it to make it look better.